database/schema.sql

```diff
@@ -1,17 +1,154 @@
 CREATE TABLE IF NOT EXISTS customers (
   ID INTEGER PRIMARY KEY AUTOINCREMENT,
+  reservation_id    INTEGER NOT NULL,
+  review_id         INTEGER NOT NULL,
   title TEXT(16),
   first_name TEXT(64) NOT NULL,
   surname TEXT (64) NOT NULL,
-  email TEXT (64)
+  email TEXT (64),
+  foreign key(reservation_id) references reservations(id),
+  foreign key(review_id)  references reviews(id)
 );
 
+CREATE TABLE invoices (
+  id                INTEGER PRIMARY KEY AUTOINCREMENT,
+  reservation_id    INTEGER NOT NULL,
+  total             INTEGER NOT NULL,
+  surcharges        TEXT (16),
+  invoice_date_time datetime,
+  paid              boolean default 0,
+  foreign key(reservation_id)  references reservations(id)
+);
+
+CREATE TABLE reservations (
+  id                INTEGER PRIMARY KEY AUTOINCREMENT,
+  customer_id       INTEGER NOT NULL,
+  room_id           INTEGER NOT NULL,
+  check_in_date     datetime,
+  check_out_date    datetime,
+  room_price        INTEGER NOT NULL,
+  foreign key(customer_id)  references customers(id),
+  foreign key(room_id)      references rooms(id)
+
+);
+
+CREATE TABLE rooms (
+  id                INTEGER PRIMARY KEY AUTOINCREMENT,
+  room_type_id      INTEGER NOT NULL,
+  sea_view          boolean,
+  foreign key(room_type_id) references room_type(id)
+);
 
-INSERT INTO customers (
+CREATE TABLE roomType (
+  id                INTEGER PRIMARY KEY AUTOINCREMENT,
+  type_name         TEXT (16),
+  orignal_price     INTEGER NOT NULL,
```

Comment:
spelling is original and it should be an number not an integer. integer is 1, 2, 3, 4, 5. number can be 323.50 - integer can't.